cypress/utils/data_utils.ts

```diff
@@ -1,21 +1,29 @@
 import * as faker from "faker";
 
-export function getStakeholderName(): string {
+export function getFullName(): string {
     return faker.name.findName();
 }
 
-export function getStakeholderEmail(): string {
+export function getEmail(): string {
     return faker.internet.email();
 }
 
-export function getBusinessServiceName(): string {
+export function getCompanyName(): string {
     return faker.company.companyName();
 }
 
-export function getBusinessServiceDescription(): string {
+export function getSentence(): string {
     return faker.lorem.sentence();
 }
 
-export function getLongString(): string {
-    return faker.random.words(75);
+export function getJobTitle(): string {
+    return faker.name.jobType();
+}
```

Comment:
Does this `jobType` completes demand of `JobTitle` which we need for our project? any example why we are specifically going with random jobType value?

Reply:
Run the test and try out the values it returns